my_list.rb

```diff
@@ -0,0 +1,16 @@
+require './my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+end
+
+list = MyList.new(1, 2, 3, 4)
+list.all? { |e| e < 5 }
+list.all? { |e| e > 5 }
+list.any? { |e| e == 2 }
+list.any? { |e| e == 5 }
+list.filter(&:even?)
```

Comment:
- [ ] When running the app nothing happens, it should return the result. Please kindly fix that

https://user-images.githubusercontent.com/74501523/201444260-f1eaa196-8d0a-4683-baa0-7d6aa9ae4b80.mov